javascript-and-json/main.js

```diff
@@ -0,0 +1,33 @@
+const bookshelf = [
+  {
+    title: 'Lord of the Rings',
+    isbn: '0562956301',
+    author: 'J.R.R. Tolkien',
+  },
+  {
+    title: 'Goodbye Eri',
+    isbn: '0989998045',
+    author: 'Tatsuki Fujimoto',
+  },
+  {
+    title: 'Holy Bible',
+    isbn: '9781414302034',
+    author: 'Jesus Christ',
+  },
+];
+
+console.log('bookshelf array:', bookshelf);
+console.log('bookshelf type of:', typeof bookshelf);
+
+const bookstring = JSON.stringify(bookshelf);
+console.log('json bookshelf:', bookstring);
+console.log('type of json bookshelf:', typeof bookstring);
+
+const jsonStudent =
+  '[{"name":"Jason","id":100230},{"name":"Mathew","id":572395}]';
```

Comment:
Create a String in JSON format holding data representing **a student** with a Number id and a String name.

This should only be one object turned into a string, not an array of objects.